src/__swaps__/screens/Swap/hooks/useSwapInputsController.ts

```diff
@@ -315,45 +351,45 @@ export function useSwapInputsController({
             type: 'outputAsset',
           },
         ].map(getAssetNativePrice)
-      );
+      ).then(([inputPrice, outputPrice]) => ({ inputPrice, outputPrice }));
     },
     [getAssetNativePrice]
   );
 
   const fetchAndUpdateQuote = async ({
```

Comment:
can rename this function as this now does more than just fetch the quote (e.g, also fetches prices)